docs/fuse-box/tutorials/api-get-price-changes-interval.mdx

```diff
@@ -0,0 +1,259 @@
+    timestamp: number
+    priceChange: string
+    previousPrice: string
+    currentPrice: string
+}
+```
+
+Add the useState Hook from React
+
+```javascript
+import { useState, useEffect } from "react";
+```
+
+Create a state to store the responses from the Fetch call using the Token interface.
+
+```javascript
+const [priceData, setPriceData] = useState<Token[]>([]);
+```
+
+Update the FetchData function by parsing the data response to the state:
+
+```javascript
+        const response = await fetch(apiUrl);
+        if (!response.ok) {
+          throw new Error('Network response was not ok');
+        }
+        const data = await response.json();
+        setPriceData(data.data);
+```
+
+To create the Chart import the required methods:
+
+```javascript
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+} from 'chart.js';
+import { Line } from 'react-chartjs-2';
+
+ ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend
+);
+```
+
+The Chart JS `Line` component takes in a series of parameters, the two most important being `options` and `data`. We are going to create an options and data variables:
+
+```javascript
+const options = {
+    responsive: true,
+    plugins: {
+      legend: {
+        position: 'top' as const,
+      },
+    },
+  };
+```
+
+```javascript
+const data = {
+    labels,
+    datasets: [
+      {
+        label: 'LadyBot Token Price Change Per Hour',
+        data: priceData.map((token) => token.currentPrice),
+        borderColor: 'rgb(255, 99, 132)',
+        backgroundColor: 'rgba(255, 99, 132, 0.5)',
+      },
+     ],
+  };
+```
+The Data object variable has `labels` as a parameter, which the y-axis and `data` going on the x-axis. As mentioned above, we will parse `timeStamp` to the y-axis and `currentPrice` to the x-axis. To do this we will map over the response stored in the `token` state. 
+
+For the `timeStamp` we will convert it to human readable date time in “Hours”, “Minutes” and “Seconds”..
+
+```javascript
+const labels = priceData.map(token => {
+      const date = new Date(token.timestamp * 1000);
+      const hours = date.getHours();
+      const minutes = date.getMinutes();
+      const seconds = date.getSeconds();
+     
+      return `${hours < 10 ? '0' : ''}${hours}:${minutes < 10 ? '0' : ''}${minutes}:${seconds < 10 ? '0' : ''}${seconds}`;
+    });
+```
+
+To display the Chart add the Line Component after the “Hello, World!” inside the `return` statement.
+
+```javascript
+<Line options={options} data={data} />
+```
+
+Save the file. Run the application using the following command:
+
+```javascript
+npm run dev
+```
+
+Visit `http://localhost:3000` in your browser to see the application in action.
+
+Checkout the complete [code](https://github.com/fuseio/examples/blob/api-price-changes-by-interval/api-price-changes-by-interval/pages/index.tsx). 💻
```

Comment:
Token type is missing in full code, and code didnt work at all, with this error:
<img width="1191" alt="Screenshot 2024-05-02 at 12 25 10" src="https://github.com/fuseio/fuse-docs/assets/68244817/ff0c8987-2c44-4c49-bb98-90aa896ae365">
, please test code by yourself before openning a PR`s

Reply:
Thank you for the feedback, @NikolayRn; I see where the error comes from. Remove the curly braces `{  }` from the `apiKey` param. Parse it directly. The code is working.

Reply:
I reinstalled node_modules and used updated code from examle repo, now it works